Spatial.tscn

```diff
@@ -184,10 +195,10 @@ __meta__ = {
 }
 
 [node name="ImportPackDialog" type="FileDialog" parent="Control/Dialogs"]
-margin_left = 289.0
-margin_top = 36.0
-margin_right = 960.0
-margin_bottom = 534.0
+margin_left = 777.0
+margin_top = 89.0
+margin_right = 1448.0
+margin_bottom = 587.0
```

Comment:
What is the motivation for moving the window?

Reply:
There are several windows that overlap each other. This moves this window to an unused space. This is something to keep in mind when we do the larger UI update.